src/components/radio-button/radio-button.styles.ts

```diff
@@ -0,0 +1,55 @@
+import { useTheme } from 'native-base';
+import { StyleSheet } from 'react-native';
+
+import { RadioButtonKind } from './radio-button.types';
+
+type RadioKind = 'primary' | 'success' | 'error';
+
+interface RadioKindStyle {
+  borderColor: string;
+  innerColor: string;
+}
+
+export const useRadioKindStyles = (): Record<RadioKind, RadioKindStyle> => {
+  const theme = useTheme();
+
+  return {
+    [RadioButtonKind.PRIMARY]: {
+      borderColor: theme.colors.primary['500'],
+      innerColor: theme.colors.primary['600'],
+    },
+    [RadioButtonKind.SUCCESS]: {
+      borderColor: theme.colors.success?.['500'] || '#16a34a',
```

Comment:
We do not want to add a fallback color here (and places below) as it makes it hard to maintain the codebase overtime especially when we customize the theme for clients.

Also `?` in `theme.colors.success?` is not desireable. It should always be present, perhaps we need to update typedef / theme

These comments are applicable for danger below as well